gemini/sample-apps/fashion-trends-prediction/app/utilities.py

```diff
@@ -0,0 +1,187 @@
+import base64
+import urllib.request
+
+from PIL import Image as ImagePIL
+from rembg import remove
+import streamlit as st
+from streamlit.components.v1 import html
+
+
+def stImg(file):
```

Comment:
Something similar is used in other files, could they also use this function?

Also make the name more specific

Reply:
The stImg function is now removed as images are now read from gcs so this is not needed.